main.cpp

```diff
@@ -58,11 +57,38 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int number;
+    int number2;
+    int output;
+    
+    
+    float gain;
+    float threshold;
+    float ratio;
+    
+    bool bypass;
+    bool systemOn;
+    bool bypassComp;
+
+    double importantProcess;
+    double freq;
+    double numbers;
+    
+ //unsure about how and why double and unsigned int 
+
+    char jam;
+    char toast;
+    char sugar; 
+//unsigned
+    unsigned int Ad;
+    unsigned int A1;
+    unsigned int Ab;
```

Comment:
You need to initialize your variables.
The example showed how to initialize an `int` type, but you deleted it for some reason.
- [ ] revise